filament/backend/include/backend/DriverEnums.h

```diff
@@ -624,6 +624,23 @@ enum class BlendFunction : uint8_t {
     SRC_ALPHA_SATURATE      //!< f(src, dst) = (1,1,1) * min(src.a, 1 - dst.a), 1
 };
 
+//! Stream for external textures
+enum class StreamType {
+    NATIVE,
+    TEXTURE_ID,
+    ACQUIRED,
+};
+
+//! Releases an ACQUIRED external texture, guaranteed to be called on the application thread.
+using StreamCallback = void(*)(void* image, void* user);
+
+//! Bundles the state of an ACQUIRED stream.
+struct AcquiredImage {
```

Comment:
I think `AcquiredImage` should be a completely opaque type -- the developper doesn't need to know what's in it.

Reply:
Good point, this was tricky though (DriverBase wasn't quite right either) so I moved this into its own header file.